### PR TITLE
deleted var

### PR DESCRIPTION
**Pull request by Abhiram-Gullapalli**
#### Related issues

Resolves issue #16662 

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by Abhiram-Gullapalli** [comment]
@teoli2003 could you review this, I think this would conclude the web/API section, although I'm pretty sure there are a few I missed out would be making a separate pull request for those,

File: files/en-us/web/api/window/getdefaultcomputedstyle/index.md [hunk 1]
```diff
@@ -83,8 +83,8 @@ pseudo-elements (e.g., {{cssxref("::before")}} or {{cssxref("::after")}}).
 <h3>generated content</h3>
 
 <script>
-  var h3       = document.querySelector('h3'),
-      result   = getDefaultComputedStyle(h3, ':after').content;
+  const h3       = document.querySelector('h3')
+  const result   = getDefaultComputedStyle(h3, ':after').content;
```
Comment (teoli2003) [inline files/en-us/web/api/window/getdefaultcomputedstyle/index.md:87]:
```suggestion
  const h3 = document.querySelector('h3');
  const result = getDefaultComputedStyle(h3, ':after').content;
```

File: files/en-us/web/api/window/location/index.md [hunk 2]
```diff
@@ -63,7 +63,7 @@ insert the value of `location.pathname` into the hash:
 
 ```js
 function reloadPageWithHash() {
-  var initialPage = location.pathname;
+  const initialPage = location.pathname;
   location.replace('http://example.com/#' + initialPage);
```
Comment (teoli2003) [inline files/en-us/web/api/window/location/index.md:67]:
```suggestion
  location.replace(`http://example.com/#${location.pathname}`);
```

File: files/en-us/web/api/window/location/index.md [hunk 3]
```diff
@@ -155,8 +155,8 @@ span.intLink {
 …the same thing but with an animated page scroll:
 
 ```js
-var showBookmark = (function () {
-  var  _useHash, _scrollX, _scrollY, _nodeX, _nodeY, _itFrame, _scrollId = -1, _bookMark,
+const showBookmark = (function () {
+  const  _useHash, _scrollX, _scrollY, _nodeX, _nodeY, _itFrame, _scrollId = -1, _bookMark,
```
Comment (teoli2003) [inline files/en-us/web/api/window/location/index.md:159]:
```suggestion
  let _useHash;
  let _scrollX;
  let _scrollY;
  let _nodeX;
  let _nodeY;
  let _itFrame;
  let _scrollId = -1;
  let _bookMark;
```

File: files/en-us/web/api/windowclient/focus/index.md [hunk 4]
```diff
@@ -46,8 +46,8 @@ self.addEventListener('notificationclick', function(event) {
   event.waitUntil(clients.matchAll({
     type: "window"
   }).then(function(clientList) {
-    for (var i = 0; i < clientList.length; i++) {
-      var client = clientList[i];
+    for (let i = 0; i < clientList.length; i++) {
+      const client = clientList[i];
```
Comment (teoli2003) [inline files/en-us/web/api/windowclient/focus/index.md:50]:
```suggestion
    for (const client of clientList) {
```

File: files/en-us/web/api/windowclient/focused/index.md [hunk 5]
```diff
@@ -35,8 +35,8 @@ self.addEventListener('notificationclick', function(event) {
   event.waitUntil(clients.matchAll({
     type: "window"
   }).then(function(clientList) {
-    for (var i = 0; i < clientList.length; i++) {
-      var client = clientList[i];
+    for (let i = 0; i < clientList.length; i++) {
+      const client = clientList[i];
```
Comment (teoli2003) [inline files/en-us/web/api/windowclient/focused/index.md:39]:
```suggestion
    for (const client of clientList) {
```

File: files/en-us/web/api/windowclient/index.md [hunk 6]
```diff
@@ -49,8 +49,8 @@ self.addEventListener('notificationclick', function(event) {
   event.waitUntil(clients.matchAll({
     type: "window"
   }).then(function(clientList) {
-    for (var i = 0; i < clientList.length; i++) {
-      var client = clientList[i];
+    for (let i = 0; i < clientList.length; i++) {
+      const client = clientList[i];
```
Comment (teoli2003) [inline files/en-us/web/api/windowclient/index.md:53]:
```suggestion
    for (const client of clientList) {
```

File: files/en-us/web/api/xmlhttprequest/using_xmlhttprequest/index.md [hunk 7]
```diff
@@ -363,8 +363,8 @@ ways to _submit_, and to **upload files**:
 
 if (!XMLHttpRequest.prototype.sendAsBinary) {
   XMLHttpRequest.prototype.sendAsBinary = function(sData) {
-    var nBytes = sData.length, ui8Data = new Uint8Array(nBytes);
-    for (var nIdx = 0; nIdx < nBytes; nIdx++) {
+    const nBytes = sData.length, ui8Data = new Uint8Array(nBytes);
```
Comment (teoli2003) [inline files/en-us/web/api/xmlhttprequest/using_xmlhttprequest/index.md:366]: nit
```suggestion
    const nBytes = sData.length;
    const ui8Data = new Uint8Array(nBytes);
```

File: files/en-us/web/api/xmlhttprequest/using_xmlhttprequest/index.md [hunk 8]
```diff
@@ -444,16 +444,16 @@ var AJAXSubmit = (function () {
   }
 
   function SubmitRequest (oTarget) {
-    var nFile, sFieldType, oField, oSegmReq, oFile, bIsPost = oTarget.method.toLowerCase() === "post";
+    const nFile, sFieldType, oField, oSegmReq, oFile, bIsPost = oTarget.method.toLowerCase() === "post";
```
Comment (teoli2003) [inline files/en-us/web/api/xmlhttprequest/using_xmlhttprequest/index.md:447]:
```suggestion
    let nFile;
    let sFieldType; 
    let oField;
    let oSegmReq;
    let oFile;
    const bIsPost = oTarget.method.toLowerCase() === "post";
```

File: files/en-us/web/api/xmlhttprequest/using_xmlhttprequest_in_ie6/index.md [hunk 9]
```diff
@@ -11,18 +11,18 @@ tags:
...
+const request = new XMLHttpRequest()
 ```
 
 However, if you need to also support Internet Explorer 6 and older, you need to extend your code like this:
 
 ```js
 if (window.XMLHttpRequest) {
     //Firefox, Opera, IE7, and other browsers will use the native object
-    var request = new XMLHttpRequest();
+    let request = new XMLHttpRequest();
```
Comment (teoli2003) [inline files/en-us/web/api/xmlhttprequest/using_xmlhttprequest_in_ie6/index.md:22]:
```suggestion
    var request = new XMLHttpRequest();
```

File: files/en-us/web/api/xmlhttprequest/using_xmlhttprequest_in_ie6/index.md [hunk 10]
```diff
@@ -11,18 +11,18 @@ tags:
...
 } else {
     //IE 5 and 6 will use the ActiveX control
-    var request = new ActiveXObject("Microsoft.XMLHTTP");
+    let request = new ActiveXObject("Microsoft.XMLHTTP");
```
Comment (teoli2003) [inline files/en-us/web/api/xmlhttprequest/using_xmlhttprequest_in_ie6/index.md:25]:
```suggestion
    var request = new ActiveXObject("Microsoft.XMLHTTP");
```

File: files/en-us/web/api/xmlhttprequest/using_xmlhttprequest_in_ie6/index.md [hunk 11]
```diff
@@ -11,18 +11,18 @@ tags:
 In all modern browsers, you can create a new XMLHttpRequest object using the following code:
 
 ```js
-var request = new XMLHttpRequest()
+const request = new XMLHttpRequest()
```
Comment (teoli2003) [inline files/en-us/web/api/xmlhttprequest/using_xmlhttprequest_in_ie6/index.md:14]: This article is about IE6. We can't use `const` and `let`.
```suggestion
var request = new XMLHttpRequest()
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

**Review by teoli2003 — APPROVED** [review]
Thank you! This is great work.

I'll double-check next week to be sure that we are done with both web/api and web/javascript, then we will move forward! (There are more javascript in other areas of MDN)

**Comment by Abhiram-Gullapalli** [comment]
Do let me know when there is a proposal for changes in another section.

**Pull request merged**